api/models/team.py

```diff
@@ -0,0 +1,29 @@
+"""Import Django models."""
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
```

Comment:
Underscore gettext here as well. See what underscore means in other languages as well:
* [Javascript](https://www.npmjs.com/package/underscore)
* [Java](https://github.com/javadev/underscore-java)
* [Perl](http://vti.github.io/underscore-perl/)

...